include/zephyr/drivers/led.h

```diff
@@ -145,7 +156,7 @@ static inline int z_impl_led_blink(const struct device *dev, uint32_t led,
 		(const struct led_driver_api *)dev->api;
 
 	if (api->blink == NULL) {
-		return -ENOSYS;
+		return led_blink_fallback(dev, led, delay_on, delay_off);
```

Comment:
I think it would be better to put this in an ifdef and implement it so that it keeps clean when it's disabled.